kuma/users/views.py

```diff
@@ -405,7 +406,11 @@ def get_context_data(self, **kwargs):
                                               .filter(reduced_or_query))
         else:
             matching_accounts = SocialAccount.objects.none()
+
+        registration_disabled = waffle.flag_is_active(self.request,
```

Comment:
This is the wrong place to put it, use the `kuma.users.adapters.KumaSocialAccountAdapter.is_open_for_signup` method to specify it. It shold automatically render a "not enabled" page.